code/modules/mob/living/simple_animal/hologram.dm

```diff
@@ -56,4 +56,101 @@
 	atom_to_mimic = /mob/living/simple_animal/cow
 
 /mob/living/simple_animal/hologram/tajaran_dancer
-	atom_to_mimic = /mob/living/simple_animal/hostile/humanoid/tajaran/dancer
\ No newline at end of file
+	atom_to_mimic = /mob/living/simple_animal/hostile/humanoid/tajaran/dancer
+
+/mob/living/simple_animal/hologram/advanced
+	name = "hologram"
+	desc = "A holographic image of a person."
+	icon_state = "holo3"
+	icon_living = "holo3"
+	held_items = list(null, null)
+	health = 50
+	maxHealth = 50
+	melee_damage_lower = 5
+	melee_damage_upper = 5
+	attacktext = "punches"
+	var/obj/item/head
+	var/obj/item/w_uniform
+	var/obj/item/wear_suit
+	var/list/obj/abstract/Overlays/obj_overlays[TOTAL_LAYERS]
+
+/mob/living/simple_animal/hologram/advanced/New()
+	..()
+	name = "[name] ([rand(1, 1000)])"
+	real_name = name
+	obj_overlays[HEAD_LAYER]		= getFromPool(/obj/abstract/Overlays/head_layer)
+	obj_overlays[UNIFORM_LAYER]		= getFromPool(/obj/abstract/Overlays/uniform_layer)
+	obj_overlays[SUIT_LAYER]		= getFromPool(/obj/abstract/Overlays/suit_layer)
+	spawn(5)
```

Comment:
If the only reason this is behind `spawn()` is to make sure the client receives it, move it to `Login()`.